src/Sulu/Bundle/ContactBundle/Controller/AccountController.php

```diff
@@ -82,12 +73,13 @@ class AccountController extends AbstractContactController
     protected $accountContactFieldDescriptors;
     protected $accountAddressesFieldDescriptors;
 
-    public function __construct()
+    public function setContainer(ContainerInterface $container = null)
```

Comment:
because it needs a set container for `getParameter`. and in the constructor the container is not set!